docs-site/content/30.0/api/README.md

```diff
@@ -27,6 +31,11 @@ This release contains important new features, performance improvements and bug f
 - Add support for Azure OpenAI models in Natural Language Search ([Docs](https://typesense.org/docs/30.0/api/natural-language-search.html#supported-model-types)).
 - Add configurable token truncation for string fields to improve exact match filtering on long strings ([Docs](https://typesense.org/docs/30.0/api/collections.html#field-parameters)).
 - Add GCP service account authentication for auto-embedding with GCP models ([Docs](https://typesense.org/docs/30.0/api/vector-search.html#service-account-authentication)).
+- Use Transliterator objects pool [PR#2412] (https://github.com/typesense/typesense/pull/2412)
```

Comment:
Mention that this helps improve indexing performance for some locales (check which use Transliterator pool).